concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -700,6 +694,38 @@ jobs:
           params:
             file: generated-concourse-secrets/concourse-secrets.yml
 
+      - task: generate-bosh-runtime-config-vars
+        config:
+          platform: linux
+          image_resource: &gov-paas-bosh-cli-v2-image-resource
+            type: docker-image
+            source:
+              repository: governmentpaas/bosh-cli-v2
+              tag: 0eff5b6a9c092f865a2b19cc4e75a3b539b82fa2
+          inputs:
+            - name: paas-bootstrap
+            - name: bosh-runtime-config-vars-store
+          outputs:
+            - name: new-bosh-runtime-config-vars-store
+          params:
+            BOSH_NON_INTERACTIVE: true
+          run:
+            path: sh
+            args:
+              - -ec
+              - |
+                bosh interpolate \
+                  --var-errs \
+                  --vars-store bosh-runtime-config-vars-store/bosh-runtime-config-vars-store.yml \
+                  paas-bootstrap/manifests/runtime-config/variables-template.yml
+
+                cp bosh-runtime-config-vars-store/bosh-runtime-config-vars-store.yml \
+                   new-bosh-runtime-config-vars-store/bosh-runtime-config-vars-store.yml
```

Comment:
Is there a reason this needs to be a separate task like this. Could it not be part of the set-runtime-configs task? It would make sense to me to have all the needed variables defined in the runtime-config manifest, and get rid of the `variables-template.yml` file. This would be more consistent with other manifests (eg the CF manifest).